manifests/bosh-manifest/bosh-manifest.yml

```diff
@@ -96,6 +96,9 @@ instance_groups:
       ssl:
         cert: (( grab secrets.bosh_director_cert ))
         key: (( grab secrets.bosh_director_key ))
+      local_dns:
+        enabled: true
+        use_dns_addresses: true
```

Comment:
Could this be related to the monitor that's failing in your dev environment?
https://app.datadoghq.com/monitors/5847510